artifactory/commands/transferconfig/utils_test.go

```diff
@@ -35,3 +36,17 @@ func TestArchiveConfig(t *testing.T) {
 		}
 	}
 }
+
+// In some versions of Artifactory, the file access.bootstrap.json has a typo in its name: access.boostrap.json.
+// This tests this scenario.
+func TestArchiveConfigWithTypoInAccessBootstrap(t *testing.T) {
```

Comment:
I think that instead of duplicating artifactory_export, maybe it would be better to export TestArchiveConfig content to an internal function testArchiveConfig(t *testing.T).
After that, we can run 2 tests:
1. `TestArchiveConfig` - Just run testArchiveConfig(t)
2. `TestArchiveConfigBoostrap` - Rename access.bootstap.json -> access.boostrap.json and run testArchiveConfig(t)